### PR TITLE
#738: Add route data to ComputedBoundsAction

### PR DESCRIPTION
**Pull request by martin-fleck-at**
Part of https://github.com/eclipse-glsp/glsp/issues/738

### CODE REVIEW

**Review by tortmayr — CHANGES_REQUESTED** [review]
Thanks Martin. Code seems to work fine. 
I have tested it by changing the routerKind of the workflow example edge creation handlers to "manhattan".

I have a few inline suggestions and one more general question:

I personally find it a bit odd that an action called "ComputedBoundsAction" also contains the routing points. Also by directly adapting the "ComputedBoundsAction" which we might introduce an inconsistency/incompatibility with the original ComputedBoundsAction from sprotty.

Maybe it would be better to use a dedicated `ComputedRoutingPointsOperation` and wrap it together with the `ComputedBoundsOperation` in an "CompoundOperation`?

File: plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/utils/LayoutUtil.java [hunk 1]
```diff
@@ -51,29 +53,60 @@ private LayoutUtil() {}
...
+      }
+   }
+
+   /**
+    * Applies the new routing points to the model.
+    *
+    * @param routingPoints The new routing points.
+    * @param index         The model index.
+    */
+   public static void applyRoutingPonts(final ElementAndRoutingPoints routingPoints, final GModelIndex index) {
```
[the remark this answers is not in the document]
Comment (martin-fleck-at) [reply plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/utils/LayoutUtil.java]: Ah, good catch, thank you!

File: plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/gmodel/GModelChangeRoutingPointsHandler.java [hunk 2]
```diff
@@ -15,16 +15,11 @@
  ********************************************************************************/
 package org.eclipse.glsp.server.gmodel;
```
[the remark this answers is not in the document]
Comment (martin-fleck-at) [reply plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/gmodel/GModelChangeRoutingPointsHandler.java:16]: Done!

**Comment by martin-fleck-at** [comment]
@tortmayr Thank you very much for your feedback, I adapted the code accordingly. Regarding a dedicated operation for the routing points, I opted against it since the client-side layout information is currently all part of the `ComputedBoundsAction`, also the alignment. Furthermore, this action carries a specific meaning as the `ComputedBoundsActionHandler` will directly submit the model afterwards, so splitting that into two operations would make their handling more difficult.

**Pull request merged**